tests/bats/integration.bats

```diff
@@ -30,7 +30,19 @@ load _helpers_govcms
   pushd "${SCAFFOLD_DIR}" || exit 1
 
   # Download PaaS scaffold to be able to bootstrap the site.
-  download_code_from_github "govCMS" "govCMS8-scaffold-paas"
+  # If the artifact can't be download we skip this test.
+  # download_code_from_github "govcms" "scaffold"
+  git clone --depth 1 https://github.com/govCMS/scaffold "$(pwd)"
```

Comment:
Does it matter that this will grab `develop` instead of `master`?

Reply:
I was contemplating using `--branch <master> --single-branch` for the shallow clone, but I think it's safe to use develop for this particular test case.

Reply:
Also, I believe you could have used `${SCAFFOLD_DIR}` here.